### ISSUE TITLE
Allow database configuration/settings path to be specified for CLI commands

### ISSUE THREAD

**Issue by ghost** [MasoniteFramework/orm#501]
**Describe the feature as you'd like to see it**
It should be possible to specify where to load settings from when running migrations, seeders, etc using the `masonite-orm` CLI

This would allow the ORM to be used/integrated in existing projects without having to rewrite the rest of the application to use the predefined configuration directory (`config/database(.py)` to avoid duplication of config/settings directories.

**What do we currently have to do now?** 
There are multiple ways to implement the loading itself but the simplest way might be adding another CLI option that allows you to specify the configuration directory.

Example command: `masonite-orm migrate --directory=src/database/migrations --config=src/settings/database`

**Additional context**
Example class/import triggered by `masonite-orm migrate` that does not currently work if settings are not in the exact specified pypath and format.
![image](https://user-images.githubusercontent.com/49159860/133235910-0b6fc6dc-4062-43de-9f51-8ff2bbdc381a.png)

This will currently result in the following module import error
```bash
(venv) C:\Users\{USER}\Code\{PROJECT}\{REPO}>masonite-orm migrate -d=src\api\database

  ModuleNotFoundError

  No module named 'config'

  at ~\Code\EVA_AST\ast_new\venv\lib\site-packages\masoniteorm\migrations\Migration.py:27 in __init__
       23│         self.migration_directory = migration_directory
       24│         self.last_migrations_ran = []
       25│         self.command_class = command_class
       26│
    →  27│         from config.database import DB
       28│
       29│         DATABASES = DB.get_connection_details()
       30│
       31│         self.schema = Schema(
```

This is already available in orator
![image](https://user-images.githubusercontent.com/49159860/133238300-823fd904-a1f4-4f45-b971-fb04563a81e1.png)

~~- [ ] Is this a breaking change?~~
**Only if the existing import/path is not set as default**

**Comment by josephmancuso** [comment]
I couldn't think of a good way to do this at the time but your issue just gave me a good idea .. We could save the default as a string and we could do something like:

```
DB = pydoc.locate('config.database.DB')
```

We could then take that as an input to this class.

That would fix the migrations issue and allow specifying the config path. The issue is that there are a few other places we hard coded this config path.

The reason this works so well for Orator is because in Orator you import the model class from the config file. We wanted to avoid that and allow you to import the model, and other classes, from the library paths. 

If we can figure out how to also avoid those then we could do this. I'm open to discuss more.

**Comment by ghost** [comment]
That could work 😄I think using a controllable parameter like a environment variable would still be preferable, but they're also not mutually exclusive? Like it could attempt to load from environment and then default to the `locate`, aka something like:
- `DB = os.getenv("DB_CONFIG_PATH", pydoc.locate('config.database.DB'))`
- `DB = os.getenv("DB_CONFIG_PATH", None) or pydoc.locate('config.database.DB'))`

🤔 try/catch/finally would also give some additional implementation options

**Comment by josephmancuso** [comment]
Yeah. You understand the problem we're trying to solve here lol. Can you open a PR for this?

**Comment by josephmancuso** [comment]
This will be available in Version 2.0 which will release with Masonite 4

**Comment by ghost** [comment]
Apologies for lack of reply, thanks for the update!
